app/bundles/CoreBundle/Helper/DateTimeHelper.php

```diff
@@ -64,16 +70,15 @@ public function setDateTime($datetime = '', $fromFormat = 'Y-m-d H:i:s', $timezo
     {
         $localTimezone = date_default_timezone_get();
```

Comment:
Here you're using `$localTimezone`, but a few lines down you're using `$this->localTimezone`. That's inconsistent. From reading the code I think the line here is superfluous.